dockerfiles/web/Dockerfile

```diff
@@ -1,20 +1,19 @@
-FROM python:3-buster
+ARG TRAVIS_PYTHON_VERSION
+FROM python:${TRAVIS_PYTHON_VERSION:-3.5}-buster
 
 WORKDIR /usr/src/app
 
 ENV DEBIAN-FRONTEND noninteractive
 ENV DISPLAY=:1
 ENV GECKODRIVER_VERSION="v0.26.0"
 RUN echo "deb http://deb.debian.org/debian stretch-backports main" > /etc/apt/sources.list.d/backports.list
-RUN apt-get update && apt-get install -y xvfb firefox-esr libpq-dev python3-dev
-RUN apt-get install -y -t stretch-backports libsqlite3-0
+RUN wget -O - https://deb.nodesource.com/setup_10.x | bash -
```

Comment:
It looks like we're also moving to a different version of Node here, in addition to changes to Python? What's up w/ that?

Reply:
Oh, that was in response to the EOL notice I saw when doing the docker build (8.x EOLed, and now 10 is the current LTS).

Reply:
What do we actually use Node for besides generating static assets? I feel like we should do a little testing

Reply:
FWIW tests are passing, but I don't know how much automated in-browser testing (e.g, via Selenium or similar) we do. I haven't done a comprehensive click-around-and-make-sure-everything-works; in particular, our image upload feature uses a lot of frontend magic IIRC, and that can't even be tested locally, only on staging (unless you set up a personal S3 bucket).

What do you think about splitting this one change out to a separate PR so that we can merge your (very necessary) Python fix, while making sure we properly QA any JavaScript that might be broken by this?